src/ray/gcs/gcs_server/gcs_actor_manager.h

```diff
@@ -336,6 +337,9 @@ class GcsActorManager : public rpc::ActorInfoHandler {
   Status RegisterActor(const rpc::RegisterActorRequest &request,
                        RegisterActorCallback success_callback);
 
+  /// Set actors on the node as preempted and publish the actor information.
```

Comment:
```suggestion
  /// Set actors on the node as preempted and publish the actor information.
  /// If the node is already dead, it will be no-op.
```